lib/rules/one-var.js

```diff
@@ -14,16 +15,39 @@
 
 module.exports = function(context) {
 
-    var MODE = context.options[0] || "always";
-    var options = {};
+    var MODE = context.options[0];
```

Comment:
I know it was like this in the original file, but this variable should be in camel case. Would you mind changing while you're in here?